tools/ci-scripts/check-aws-sdk-standalone-integration-tests

```diff
@@ -39,4 +39,11 @@ find "${tmp_dir}"
 
 pushd "${tmp_dir}/aws/sdk/integration-tests"
 cargo check --tests --all-features
+
+# Running WebAssembly (WASI) specific integration tests
+pushd "${tmp_dir}/aws/sdk/integration-tests/webassembly" &>/dev/null
+cargo check --tests
+RUST_TEST_NOCAPTURE=1 cargo component test
```

Comment:
This script isn't intended to actually run any tests since they run in a separate script. This one just checks that we didn't break the dev-only Cargo.toml files in aws/sdk/integration-tests, so it only needs `cargo check`

Reply:
Removed the `test` invocation here